.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
I don't know if this file is redundant or not. I left similar comments on the sibling PR.

Reply:
keeping this file and removing the husky stuff from package.json